Src/Base/AMReX_ParmParse.H

```diff
@@ -1192,14 +1192,16 @@ public:
      */
     template <typename T, typename ET = amrex_enum_traits<T>,
               std::enable_if_t<ET::value,int> = 0>
-    int query (const char* name, T& ref)
+    int query (const char* name, T& ref, int ival = FIRST) const
     {
         std::string s;
-        int exist = this->query(name, s);
+        int exist = this->query(name, s, ival);
         if (exist) {
             try {
                 ref = amrex::getEnum<T>(s);
             } catch (...) {
+                amrex::Print() << "amrex::ParmParse::query (input name: "
```

Comment:
We need to allow the users to turn off warning messages. So we either remove this, or put this inside `if (amrex::Verbose() > xxx)`, or add the message to what is being thrown by amrex::getEnum.

Reply:
Done.